### PR TITLE
Dropdown bugfix

### PR DESCRIPTION
**Pull request by kaitj**
Addresses #52 - Selected template not displayed following selection from dropdown due to font color.

Changed font color to white and dropdown to black. More discussion / planning needed and will be addressed in #47, but this will at least fix the validator in its current state.

### CODE REVIEW

**Comment by Park-Patrick** [comment]
looks good to me!

**Comment by Park-Patrick** [comment]
we should probably also change the part in the README about the submodules as well

**Comment by kaitj** [comment]
> we should probably also change the part in the README about the submodules as well

Agreed and it has been updated #57. I realized I didn't exactly make it easy to make the merge as I based this off of #57 instead of #57. 

Ideally, #55 gets pulled in first to enable automatic reviewer assignment and then we can merge both #57 and #58.

File: templates/view.html [hunk 1]
```diff
@@ -19,11 +19,11 @@
       <h2>Setup</h2>
       <form method=post action="" enctype="multipart/form-data">
 
-        <fieldset>
+        <fieldset style>
```
Comment (tkkuehn) [inline templates/view.html]: What does this do?

Comment (kaitj) [reply templates/view.html]: It sets the background color of the dropdown menu to black and the font color to white. The proper way is probably to implement in css, but this was a quick fix to the current problem.

Comment (tkkuehn) [reply templates/view.html]: It makes sense to me that the `style` attributes on the `select` and `option` elements do that, but does the empty `style` attribute on the `fieldset` element do anything?

Comment (kaitj) [reply templates/view.html]: Oh, I see what you're pointing to now. Might have been a little oopsie when I was just trying to get that fixed. Anyways, it is now updated and pushed!

**Comment by tkkuehn** [comment]
@kaitj Needs a merge from master, will merge once that's done.

**Comment by kaitj** [comment]
Done!

**Pull request merged**